hathor/p2p/states/ready.py

```diff
@@ -178,7 +175,6 @@ def handle_peers(self, payload: str) -> None:
         received_peers = json_loads(payload)
         for data in received_peers:
             peer = Peer.create_from_json(data)
-            peer.validate()
```

Comment:
Why don't you have to validate anymore?

Reply:
Because a `Peer` instance doesn't have a `validate` method anymore, since all the validations are only related to public and private keys, and thus only `PublicPeer` and `PrivatePeer` have a `validate` method. In this case the validation method that was removed had no effect.

Reply:
For my education, what kind of validation was performed before?

Reply:
Got it. Thanks!